app/views/comments/_form.html.erb

```diff
@@ -66,37 +65,64 @@
 
   <%= javascript_include_tag "dragdrop" %>
   <script>
-    $('#comment-form').bind('ajax:beforeSend', function(event){      
+    $('#comment-form').bind('ajax:beforeSend', function(event){
       $("#text-input").prop('disabled',true)
-      $("#comment-form .btn-primary").button('loading',true);      
+      $("#comment-form .btn-primary").button('loading',true);
     });
-    <% emoji_names = emoji_names_list %>
-    <% emojis = emoji_names[:emojis] %>
-    <% image_map = emoji_names[:image_map] %>
-    var image_map = <%= raw image_map.to_json %>
-    horsey(document.querySelector('textarea#text-input'), {
-        suggestions: (<%= raw emojis.to_json %>),
-        anchor: ':',
-        render: function (li, suggestion) {
-            var image = image_map[suggestion.text];
-            li.innerHTML = "<b>"+image+"</b><span>"+suggestion.text+"</span>";
+
+    var at_config = {
+      at: "@",
+      callbacks: {
+        remoteFilter: function(query, callback) {
+          $.getJSON("/api/srch/profiles?srchString=" + query, {}, function(data) {
+            if (data.hasOwnProperty('items') && data.items.length > 0) {
+              callback(data.items.map(function(i) { return i.docTitle }));
+            }
+           });
+          }
         },
         limit: 6
-    });
+      },
+      hashtags_config = {
+        at: "#",
+        callbacks: {
+          remoteFilter: function(query, callback) {
+            $.post('/tag/suggested/____' + query, function(response) {
```

Comment:
Ah, oops, just need to remove this `____` here, and let it say just "query" - thanks!